linera-views/src/batch.rs

```diff
@@ -72,6 +72,20 @@ pub struct SimpleUnorderedBatch {
     pub insertions: Vec<(Vec<u8>, Vec<u8>)>,
 }
 
+impl SimpleUnorderedBatch {
+    /// Printing the simple unordered batch
+    pub fn print(&self) {
```

Comment:
Please let's use the `Debug` trait (as usual in Rust) for printing values during debugging.

Reply:
I tended to like this more than the Debug but I will follow your advice and remove it.